description: always use the icognito dark theme
author: phR0ze

--- a/ui/views/window/frame_background.cc	2017-11-13 13:09:58.000000000 -0700
+++ b/ui/views/window/frame_background.cc	2017-11-22 12:31:23.229510581 -0700
@@ -17,7 +17,7 @@
     : frame_color_(0),
       use_custom_frame_(true),
       is_active_(true),
-      incognito_(false),
+      incognito_(true),
       top_area_height_(0),
       left_edge_(nullptr),
       top_edge_(nullptr),
--- a/ui/views/window/frame_background.h	2017-11-13 13:09:58.000000000 -0700
+++ b/ui/views/window/frame_background.h	2017-11-22 12:31:10.069511100 -0700
@@ -37,7 +37,7 @@
   void set_is_active(bool is_active) { is_active_ = is_active; }
 
   // Sets whether the frame to be drawn is in incognito mode.
-  void set_incognito(bool incognito) { incognito_ = incognito; }
+  void set_incognito(bool incognito) { incognito_ = true; }
 
   // Sets the theme image for the top of the window.  May be null (empty).
   // Memory is owned by the caller.
